README.md

```diff
@@ -0,0 +1,10 @@
+
+
+Mining Server Proxy
+=====================================
+
+[![Build Status](https://travis-ci.org/TheBlueMatt/mining-proxy.svg?branch=master)](https://travis-ci.org/TheBlueMatt/mining-proxy)
```

Comment:
Ugh, I hate these things, no external non-plaintext-readable crap, please.

Reply:
fixed in latest commits